src/Microsoft.SqlTools.ServiceLayer/LanguageServices/PeekDefinition.cs

```diff
@@ -346,6 +272,71 @@ internal StringCollection GetStoredProcedureScripts(string sprocName, string sch
             }
         }
 
+        #region Helper Methods
+        /// <summary>
+        /// Return schema name from the full name of the database. If schema is missing return dbo as schema name.
+        /// </summary>
+        /// <param name="fullObjectName"> The full database qualified name(database.schema.object)</param>
+        /// <param name="objectName"> Object name</param>
+        /// <returns>Schema name</returns>
+        internal string GetSchemaFromDatabaseQualifiedName(string fullObjectName, string objectName)
+        {
+            if(!String.IsNullOrEmpty(fullObjectName))
```

Comment:
please use `string` instead of `String` in all cases. This is best practice in all our code.